common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -194,9 +194,15 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
 
 
   @markdownToXml: (markdown)->
-    toXml = `function (markdown) {
+    toXml = `function (markdown) {  
       var xml = markdown,
-          i, splits, scriptFlag;
+          i, splits, scriptFlag, pTag;
```

Comment:
i here is a duplicate declaration. it's also been declared at line 218 in for statement
`for (i = 0; i < options.length; i += 1) {`